test/test-template-engine.js

```diff
@@ -354,16 +354,29 @@ describe('html rendering', () => {
     assert(container.querySelector('#target').prop === 'bar');
   });
 
+  it('renders “on*” attributes as event handlers', () => {
```

Comment:
Thought it was worth adding related coverage to `on*` _attributes_ usage (versus just properties). The interface is subtly different (and confusing) in that you more-or-less are binding the function _content_ with the _attribute_ and a function _reference_ with the _property_. And, because properties cannot be literal strings in our DSL, there’s an extra layer of nuance.

To lock this down… I just added an extra test. Why not.